### PR TITLE
Selenium: stabilize selenium tests, add tests to UNDER_REPAIR group

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR:
- increase timeout for checking that workspace from **JavaTheiaOpenshift** stack started(```CreateWorkspaceFromJavaTheiaOpenshiftStackTest``` test);
- add info about known #11697 issue to ```ContexMenuEditorTest```;
- add ```CreateAndDeleteProjectsTest``` and ```AutocompleteProposalJavaDocTest``` selenium tests to UNDER_REPAIR group.
- add info about known #12220 issue to selenium tests from ```testrunner``` package;
- add info about known #12218 issue to **CreateWorkspaceFromJavaTheiaOpenshiftStackTest** and **CreateWorkspaceFromJavaTheiaDockerStackTest** tests.

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1318/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1318//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12213
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1322/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1322//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12213
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by riuvshin** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java [hunk 1]
```diff
@@ -78,33 +76,27 @@
...
         ws.getId(), Paths.get(resource.toURI()), PROJECT, ProjectTemplates.CONSOLE_JAVA_SIMPLE);
 
     ide.open(ws);
-    loader.waitOnClosed();
+    ide.waitOpenedWorkspaceIsReadyToUse();
+
     projectExplorer.waitItem(PROJECT);
     notifications.waitProgressPopupPanelClose();
-    runCompileCommandByPallete(compileCommand);
+    runCompileCommand();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java]: What about checking running the test from command pallet?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java [hunk 2]
```diff
@@ -98,13 +93,13 @@ public void prepareTestProject() throws Exception {
     notifications.waitProgressPopupPanelClose();
     projectExplorer.quickExpandWithJavaScript();
 
-    runCompileCommandByPallete(compileCommand);
+    runCompileCommand();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java]: Could you, please, explain, why verification the command palette had been removed?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4CheckRunSuitesAndScopesTest.java [hunk 3]
```diff
@@ -72,9 +63,10 @@ public void prepareTestProject() throws Exception {
         JUNIT4_PROJECT,
         ProjectTemplates.CONSOLE_JAVA_SIMPLE);
     ide.open(ws);
-    loader.waitOnClosed();
+    ide.waitOpenedWorkspaceIsReadyToUse();
+
     projectExplorer.waitItem(JUNIT4_PROJECT);
-    runCompileCommandByPallete(compileCommand);
+    runCompileCommand();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4CheckRunSuitesAndScopesTest.java]: Could you, please, explain, why verification the command palette had been removed?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java (hunk 2 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java]: Sometimes we have problems with command creation by ```TestCommandServiceClient```(https://github.com/eclipse/che/blob/029643104cb3b8555a068bc12082b583ebdbd565/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java#L83) and the created command is not visible in Command Pallete(see screenshot).
org eclipse che selenium testrunner javatestplugintestngtest preparetestproject_time-1544721265452-millis

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1327/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1327//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12213
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**